build.savant

```diff
@@ -64,12 +66,12 @@ groovyTestNG.settings.groovyVersion = "4.0"
 groovyTestNG.settings.javaVersion = "17"
 idea.settings.imlFile = "idea-plugin.iml"
 idea.settings.moduleMap = [
-    "org.savantbuild:savant-core:2.0.0-RC.6"                  : "savant-core",
-    "org.savantbuild:savant-dependency-management:2.0.0-RC.6" : "savant-dependency-management",
-    "org.savantbuild:savant-io:2.0.0-RC.6"                    : "savant-io",
-    "org.savantbuild:savant-utils:2.0.0-RC.6"                 : "savant-utils",
-    "org.savantbuild:savant-version:2.0.0-RC.6"               : "savant-version",
-    "org.savantbuild.plugin:dependency:2.0.0-RC.6"            : "dependency-plugin"
+    ("org.savantbuild:savant-core:${savantVersion}".toString())                 : "savant-core",
```

Comment:
Same comment about using the version in the string.

Reply:
Reverted after talking w/ Daniel.